src/components/SearchResult/SearchResult.scss

```diff
@@ -17,6 +22,10 @@
     box-shadow: none;
   }
 
+  &:focus{
+    outline: none;
+  }
+
```

Comment:
`button-reset` already removes outline when not using keyboard nav, but ensures that it exists when user needs to see it to track their keyboard focus. Removing this here will lower accessibility, recommend removing.

```suggestion
```

Reply:
Previous implementation did not have the focus outline. I'll remove this

Reply:
done